source/test/TestApp/TestApp.cpp

```diff
@@ -89,17 +103,21 @@ namespace PlayFabUnit
             std::bind(&TestApp::OnPostReportError, this, std::placeholders::_1, std::placeholders::_2),
             &testRunner.suiteTestReport);
 
-        while (cloudResponse.empty())
+        for (int i = 0; i < CLOUDSCRIPT_TIMEOUT_MS; i+= CLOUDSCRIPT_TIMEOUT_INCREMENT)
```

Comment:
>+= CLOUDSCRIPT_TIMEOUT_INCREMENT [](start = 53, length = 32)

it may help clarity to place a compile-time assert that CLOUDSCRIPT_TIMEOUT_MS is evenly divisible by CLOUDSCRIPT_TIMEOUT_INCREMENT

`std::static_assert(CLOUDSCRIPT_TIMEOUT_MS  % CLOUDSCRIPT_TIMEOUT_INCREMENT == 0)`

note: this should not be impacted by your larger "how to do asserts" work item since this is a compile time assert and should not affect customers at runtime (also it's just in the test app)

Reply:
Added a comment to the assert task.